lib/Parser/Parse.cpp

```diff
@@ -8567,6 +8567,18 @@ void Parser::DeferOrEmitPotentialSpreadError(ParseNodePtr pnodeT)
     }
 }
 
+bool Parser::IsTerminateToken()
+{
+    return (m_token.tk == tkRCurly ||
+        m_token.tk == tkRBrack ||
+        m_token.tk == tkRParen ||
+        m_token.tk == tkSColon ||
+        m_token.tk == tkColon ||
+        m_token.tk == tkComma ||
```

Comment:
I see that `tkComma` was also missing before; is it correct that it should have been included as well?

Reply:
It was already there. look at the line 9069.

Reply:
Ah, I had indeed missed that.